src/system/version_migration/versions/v1_0_4.ts

```diff
@@ -46,12 +46,14 @@ export const systemMigrators = [
    * @param data {@linkcode SystemSaveData}
    */
   function fixStarterData(data: SystemSaveData) {
-    for (const starterId of defaultStarterSpecies) {
-      if (data.starterData[starterId]?.abilityAttr) {
-        data.starterData[starterId].abilityAttr |= AbilityAttr.ABILITY_1;
-      }
-      if (data.dexData[starterId]?.caughtAttr) {
-        data.dexData[starterId].caughtAttr |= DexAttr.FEMALE;
+    if (data.starterData !== null) {
```

Comment:
```suggestion
    if (!isNullOrUndefined(data.starterData)) {
```
Actually thinking about it, maybe it should be this?

Will require the import as well (also replace all those `../..` in the other imports lol).